extensions/applicationinsights-cfgsync-js/src/CfgSyncPlugin.ts

```diff
@@ -317,7 +317,9 @@ export class CfgSyncPlugin extends BaseTelemetryPlugin implements ICfgSyncPlugin
                         if (JSON) {
                             let cdnCfg = JSON.parse(response); //comments are not allowed
                             let cfg = applyCdnfeatureCfg(cdnCfg, _self.core);
-                            cfg && _setCfg(cfg, isAutoSync);
+                            let newCfg = cfg && isPlainObject(cfg) && _replaceTartgetByKeys(cfg);
+                            newCfg && _setCfg(newCfg, isAutoSync);
```

Comment:
previous design: any configs from cdn, they will be directly applied to current `core.config` (it was designed to use `blkCdnCfg` to control if we need to apply cdn changes or not).
Current:
So add an extra layer, even if `blkCdnCfg` is set to `false`, any config values defined in `nonOverrideConfigs`  will not be replaced by cdn configs.